src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004_With_LAPS.ps1

```diff
@@ -3,25 +3,27 @@
 
 <#
     .DESCRIPTION
-    Applies CIS Level one benchmarks for Windows 10 build 2004 with no exclusions.
+    Applies CIS Level one benchmarks for Microsoft_Windows_10_Enterprise build 2004 the no exclusions.
```

Comment:
was this 'with' supposed to become 'the'?

Reply:
Looks like the plaster template is wrong.

Reply:
latest push fixes that on all the examples. Plaster fix is on #157 which reset your approval there.

Reply:
I know there are under scores in the OS name also. I need to open an issue for it. The fix is rather tedious to impliment.

Reply:
I figured the underscores were an artistic choice.